pkgs/applications/blockchains/bitbox/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildNpmPackage
+, clang
+, go
+, qtwebengine
+, wrapQtAppsHook
+}:
+
+let
+  src = fetchFromGitHub {
+    owner = "digitalbitbox";
+    repo = "bitbox-wallet-app";
+    rev = "v${version}";
+    fetchSubmodules = true;
+    hash = "sha256-kvkfz9zwleZNB+leefx+cciJCPkFSSRDxBNRKNqeODc=";
+  };
+  version = "4.40.0";
+  web = buildNpmPackage {
+    pname = "bitbox-web";
+    inherit version;
+    inherit src;
+    sourceRoot = "source/frontends/web";
+    npmDepsHash = "sha256-bnMmeSX8UZpndHK0NMLQhX1GSunO0JSyZdUTfG+rSpY=";
+    installPhase = "cp -r build $out";
+  };
+in
+stdenv.mkDerivation {
+  pname = "bitbox";
+  inherit version;
+
+  inherit src;
+
+  patches = [
+    ./desktop.patch
+    ./genassets.patch
+  ];
+
+  dontConfigure = true;
+
+  buildPhase = ''
+    ln -s ${web} frontends/web/build
```

Comment:
please add a `passthru.web` attribute and use that instead of web

Reply:
`web` here is a separate derivation because I want to use `buildNpmPackage` to build what's inside `frontends/web`.

Reply:
yes and you can still do that if you add web as a passthru and use that here, that would allow usesers to override the web derivation, to for example change the version. If you need a example of what I mean look at the alice-lg package

Reply:
I got it now, ef353ff1dfb1be7052ba92e454a5eaa81cbef93b.
Great suggestion! Thank you!